### PR TITLE
crude for Logs #22

### PR DESCRIPTION
**Pull request by igorkraskovskiii**

### CODE REVIEW

File: src/main/java/app/dao/impl/LogsDaoImpl.java [hunk 1]
```diff
@@ -1,55 +1,70 @@
...
 
-    @Override
-    public synchronized void save(Logs logs) {
-        logsList.add(logs);
+    public void save() {
+        try {
+            preparedStatement.setInt(1, 2);
+            preparedStatement.setInt(2, 3);
+            preparedStatement.setInt(3, 8);
+            preparedStatement.setString(4, "Добавить рисунок");
```
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/LogsDaoImpl.java:41]: hardcoded values? you should take them from your rest serves

File: src/main/java/app/dao/impl/LogsDaoImpl.java [hunk 2]
```diff
@@ -1,55 +1,70 @@
...
         }
-        return resultLogsList;
     }
 
-    @Override
-    public synchronized List<Logs> getAll() {
-        return logsList;
+    public String getAll() {
+        Logs logs = new Logs();
+        String query = "select*from logs";
```
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/LogsDaoImpl.java]: magic string move it constant

File: src/main/java/app/dao/impl/LogsDaoImpl.java [hunk 3]
```diff
@@ -1,55 +1,70 @@
...
-        Logs previous;
-        while (logsIterator.hasPrevious()) {
-            previous = logsIterator.previous();
-            if (previous.getDate().before(boundaryDate)) {
-                break;
-            }
-            resultLogsList.add(previous);
+    public LogsDaoImpl() {
+        try {
+            connection = DriverManager.getConnection(HOST, USERNAME, PASSWORD);
```
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/LogsDaoImpl.java]: where you close connection?

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/dao/impl/LogsDaoImpl.java (hunk 2 above)
Comment (igorkraskovskiii) [reply src/main/java/app/dao/impl/LogsDaoImpl.java]: corrected

File: src/main/java/app/dao/impl/LogsDaoImpl.java (hunk 3 above)
Comment (igorkraskovskiii) [reply src/main/java/app/dao/impl/LogsDaoImpl.java]: corrected

File: src/main/java/app/dao/impl/LogsDaoImpl.java (hunk 1 above)
Comment (igorkraskovskiii) [reply src/main/java/app/dao/impl/LogsDaoImpl.java:41]: corrected

**Pull request merged**